docs/source/model_design/vivarium_model_components/risk_factors/risk_mediation/index.rst

```diff
@@ -30,4 +30,399 @@ Risk factor mediation
+Crucially, in this case, the RR of SBP→IHD **must be adjusted for BMI** since
+that is a confounder of this effect.
+
+Note that while the ideal would be evidence from a randomized controlled trial,
+therefore controlling for *all* confounders (known and unknown), that frequently isn't possible,
+including in this example:
+there isn't a way to randomly assign people BMIs or SBPs since these variables
+are not directly manipulable.
+
+GBD-modeled mediation
++++++++++++++++++++++
+
+.. todo::
+  This section only covers mediation methods from GBD 2019.
+  It appears that substantial aspects of mediation may change in GBD 2021.
+  We have yet to model any mediation using GBD 2021 but will need to expand this
+  to cover new methods.
+
+In a mediation situation where the primary risk and mediator are both GBD-modeled risks
+and the outcome is a GBD-modeled cause, GBD may report a "mediation factor" for the pathway.
+(If it does not, that means that one or more of the causal effects in the mediation diagram
+were not modeled by GBD.)
+Mediation factors for GBD 2019 and 2021 can be found `on this HUB page <https://hub.ihme.washington.edu/display/GBD2020/GBD+2021+Risk+factors+mediation>`_.
+GBD mediation factors do not vary by location, age, sex, or year.
+They are not allowed to exceed 1; if the calculated value is greater than 1 it
+will be truncated to exactly 1 (this truncation happens at the draw level).
+Some mediation pathways have a mediation factor of exactly 1 by definition, because
+there is no direct effect.
+For example, lead exposure's effect on ischemic heart disease is entirely mediated by
+high systolic blood pressure.
+Draws of the mediation factor are allowed to be negative, which represents the direct pathway
+and the mediated pathway having different effect directions.
+However, if more than 50 of the 1,000 draws were negative, that mediation
+pathway was excluded from GBD.
+
+.. note::
+  Excluding when more than 50 of the 1,000 draws are negative is a kind of "significance" test.
+  However, it assumes that a significant *negative* mediation factor
+  (direct and mediated pathways have opposite effect directions) could never occur;
+  therefore GBD never models such mediation situations.
+
+This "mediation factor" can be used to calculate values for each of the causal effects represented by arrows
+in the diagram in one of two ways; the following sections describe how to do this.
+
+Unfortunately, GBD mediation factors are inconsistent with the multiplicative application of RRs.
+This is because mediation factors are defined as proportions of the *excess* risk -- the amount the RR is
+greater than 1.
+In effect, GBD's mediation methods can split e.g. an RR of 2.2 into an RR of 1.7 and an RR of 1.6 (since 0.7 + 0.6 = 1.2, which
+is the overall RR's excess risk).
+However, if we apply these two RRs, we end up with an RR of of 1.7 * 1.6 = 2.72, which is higher than
```

Comment:
What is the causal diagram here? Is it R1 -> R2 -> O, without any direct effect? Or is it something else? I think it would be good to include a picture to show what's going on.

Reply:
Also, what are the mediation factors in this example?

Reply:
This was intended to be about a single mediator, where there is both a direct and a mediated effect. I should be more explicit about this. If 1.7 was the RR _through_ the mediator and 1.6 was the _direct_ RR, the mediation factor here would be `0.7 / 1.3 = 0.538`.